playbooks/roles/caasp4/templates/inventory-caasp4.yml.j2

```diff
@@ -2,33 +2,55 @@
 caasp-masters:
   hosts:
 {% for master_ip in _terraform_json_output.ip_masters.value %}
-    {{ master_ip }}
+    caasp-master-{{ loop.index }}:
+      ansible_host: {{ master_ip }}
 {% endfor %}
+  vars:
+    ansible_user: sles
```

Comment:
this look very image dependent, but I guess we are only carying about the CI use cases for now, so we can improve later.

I expect we could do a ci inventory, whose content contains CI only vars.